src/Simplex/Chat.hs

```diff
@@ -4202,6 +4219,7 @@ chatCommandP =
       "/_app activate" $> APIActivateChat,
       "/_app suspend " *> (APISuspendChat <$> A.decimal),
       "/_resubscribe all" $> ResubscribeAllConnections,
+      "/_file_paths " *> (APISetFilePathsConfig <$> jsonP),
```

Comment:
this is confusingly similar to the other one... Maybe better make /_set file paths

Or maybe it's better to just have two commands, not sure what's the point of combining it into one, and still keep the other one.

Reply:
In which case it would be just SetTempFolder (/_temp_folder)